sass/_variables.scss

```diff
@@ -12,7 +12,7 @@ $light-text-color: #A8ABA1 !default;
 $border-color: #CCC !default;
 $icon-color: #114327 !default;
 $save-button-color: #436F64 !default;
-$chat-textarea-height: 70px !default;
+$chat-textarea-height: 100px !default;
```

Comment:
Ok, but you can still set a different height only for the case where the button is actually there (by having a special class on the textarea).